frontend/simple/sass/_groupincome.scss

```diff
@@ -126,10 +126,28 @@
 .button-box {
   display: flex;
   align-items: center;
+  flex-direction: column;
   justify-content: center;
   height: 30%;
   margin: 30px 0 0 0;
 }
 .centered-button {
   max-width: 50%;
 }
+@font-face {
+  font-family: Foco;
+  src: url(../fonts/FocoBold.ttf);
+}
+.brand-font {
```

Comment:
The word brand has little to do with products but unique marking. Its named that way because the actual font might change as "branding" often does. I do agree changing to foco might be more straight-forward.